src/scripts/payout_fix_2023_04_18.py

```diff
@@ -0,0 +1,105 @@
+"""
+For the Accounting Period April 11 - 18, 2023 there was a bug
+in the backend services attributing incorrect solver reward values
+for a substantial number of market orders:
+
+cf. <INCLUDE LINK TO BUG AND FIX>
+
+This script
+- fetches the difference between ETH & COW Payments made before and after the fix,
+- takes the difference (ignoring what is owed to us) and
+- constructs a CSV transfer file making up for anything we owe solvers.
+"""
+import os
+import pandas
+from dune_client.client import DuneClient
+from dune_client.types import Address
+
+from src.constants import COW_TOKEN_ADDRESS
+from src.fetch.transfer_file import manual_propose
+from src.models.accounting_period import AccountingPeriod
+from src.models.token import Token
+from src.models.transfer import Transfer
+
+NUMERICAL_COLUMNS = [
+    "eth_transfer",
+    "cow_transfer",
+]
+
+if __name__ == "__main__":
+    # Saved query Executions. Note that before is no longer
+    # re-constructable since the query is based on changed data.
+    EXEC_ID_BEFORE = "01GYFEPH4XV8QJTZ2QSEAE3VWK"
+    EXEC_ID_AFTER = "01GYJ7RF07BVWW5SNGYTWYFMG5"
+
+    dune = DuneClient(os.environ["DUNE_API_KEY"])
+
+    before = pandas.read_csv(dune.get_result_csv(EXEC_ID_BEFORE).data)
+    after = pandas.read_csv(dune.get_result_csv(EXEC_ID_AFTER).data)
+
+    for number_col in NUMERICAL_COLUMNS:
+        before[number_col] = before[number_col].replace("<nil>", 0)
+        after[number_col] = after[number_col].replace("<nil>", 0)
+        before[number_col] = pandas.to_numeric(before[number_col])
+        after[number_col] = pandas.to_numeric(after[number_col])
```

Comment:
Nit: Can be wrapped in another for loop iterating over data frames to reduce the number of lines.